src/mpyl/utilities/repo/__init__.py

```diff
@@ -315,15 +313,8 @@ def push(self):
     def checkout_branch(self, branch_name: str):
         self._repo.git.switch(branch_name)
 
-    def does_branch_exist(self, branch_name: str, remote=False) -> bool:
-        self._repo.remote().fetch()
-        found_branches = [
-            branch.strip(" ").strip("*")
-            for branch in self._repo.git.branch(
-                "--list", "-r" if remote else None
-            ).splitlines()
-        ]
-        return (f"origin/{branch_name}" if remote else branch_name) in found_branches
+    def does_branch_exist(self, branch_name: str) -> bool:
```

Comment:
I can't check from my phone, but I thought this function was also used to check if local branches exist?